### PR TITLE
ref(relocation): Minor copy improvements

### PR DESCRIPTION
**Pull request by azaslavsky**
The most relevant changes are seen in this image:

Screenshot 2024-04-10 at 16 17 18

I made the "understand this command" section more paragraph based and increased the line height a bit to make it easier to read. I've also made some small copy changes to the other instructions.

### CODE REVIEW

File: static/app/views/relocation/uploadBackup.tsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ export function UploadBackup({onComplete}: StepProps) {
       >
         <p>
           {t(
-            'Nearly done! The file is being uploaded to sentry for the relocation process. You can close this tab if you like. We will email  when complete.'
+            'Nearly done! Just upload your tarball here, and well send you an email when everything is ready to go!'
```
Comment (hubertdeng123) [inline static/app/views/relocation/uploadBackup.tsx:135]:
```suggestion
            'Nearly done! Just upload your tarball here, and we'll send you an email when everything is ready to go!'
```

File: static/app/views/relocation/encryptBackup.tsx [hunk 2]
```diff
@@ -44,35 +46,32 @@ export function EncryptBackup(props: StepProps) {
...
           <mark>{'--encrypt-with /sentry-admin/key.pub'}</mark>
-          {t('Encrypts the export with the public key created in the last step.')}
-        </p>
-        <p>
-          <mark>{'/sentry-admin/file.tar'}</mark>
+          {t('encrypts the data using our public key, and ')}
+          <mark>{'/sentry-admin/export.tar'}</mark>
           {t(
-            'Writes the export file into the same directory where the public key file is located.'
+            'is the name of the output tarball. This is what youll upload in the next step.'
```
Comment (hubertdeng123) [inline static/app/views/relocation/encryptBackup.tsx:68]:
```suggestion
            'is the name of the output tarball. This is what you'll upload in the next step.'
```

Comment (azaslavsky) [reply static/app/views/relocation/encryptBackup.tsx]: Ditto.

File: static/app/views/relocation/uploadBackup.tsx (hunk 1 above)
Comment (azaslavsky) [reply static/app/views/relocation/uploadBackup.tsx]: I think our lints or something similar remove these :/. Fixed.

**Review by hubertdeng123 — APPROVED** [review]

**Pull request merged**